README.md

```diff
@@ -10,6 +10,11 @@ Toggle terminal with coc.nvim. Inspired by [repl with coc.nvim](https://gist.git
 
 `<Plug>(coc-terminal-toggle)`: toggle terminal show/hide
 
+## Configurations
+
+- `terminal.REPLMappings`: association overrides between file types and the
+    command called for their REPL, default: `{}`
```

Comment:
Default is not {}.

Reply:
oh! my bad that's true. I was looking at the default value in index.ts but that's just the fallback. I had missed the values in packages.json. I'll adjust this